pkg/build/generic/docker.go

```diff
@@ -0,0 +1,100 @@
+package generic
```

Comment:
With the new package structure I proposed above, this file can be in the `pkg/build` flat package, and be called `docker_generic.go`.